src/components/people/PersonDetail/index.tsx

```diff
@@ -1,54 +1,132 @@
 import React, { useEffect } from 'react';
 import styles from './styles.less';
-import { PersonDetails, usePersonDetails } from '@equinor/fusion';
+import { PersonDetails, usePersonDetails, PersonPresence, useApiClients } from '@equinor/fusion';
 import { useState } from 'react';
+import {
+    PersonPhoto,
+    PersonPresenceIcon,
+    AccountTypeIcon,
+    SkeletonBar,
+} from '@equinor/fusion-components';
 
 export type PersonDetailProps = {
     personId?: string;
     person?: PersonDetails;
 };
 
-const getDefaultPerson = (): PersonDetails => ({
-    azureUniqueId: 'string',
-    name: 'No person assigned',
-    mail: 'noname@equinor.com',
-    jobTitle: 'www',
-    department: 'string',
-    mobilePhone: 'string',
-    officeLocation: 'string',
-    upn: 'string',
-    accountType: 'Employee',
-    company: { id: 'id', name: 'name' },
-});
-
 export default ({ personId, person }: PersonDetailProps) => {
-    const [currentPerson, setCurrentPerson] = useState<PersonDetails>(getDefaultPerson());
+    const [currentPerson, setCurrentPerson] = useState<PersonDetails | null>(null);
+    const [presence, setPresence] = useState<PersonPresence | null>(null);
+    const [isFetchingPresence, setIsFetchingPresence] = useState<boolean>(false);
+    const [presenceError, setPresenceError] = useState<Error | null>(null);
+    const apiClients = useApiClients();
 
-    const { error, personDetails } = personId
+    const { error, personDetails, isFetching } = personId
         ? usePersonDetails(personId)
-        : { error: null, personDetails: person };
+        : { error: null, personDetails: person, isFetching: false };
 
     useEffect(() => {
         if (!error && personDetails) {
             setCurrentPerson(personDetails);
-        } else if (error) {
-            setCurrentPerson(getDefaultPerson());
         }
     }, [error, personDetails]);
 
+    const fetchPresenceStatus = React.useCallback(async (mail: string) => {
+        setIsFetchingPresence(true);
+        setPresenceError(null);
+
+        try {
+            const response = await apiClients.people.getPresenceAsync(mail, '1.0-preview');
+            setPresence(response.data);
+        } catch (e) {
+            setPresenceError(e);
+        } finally {
+            setIsFetchingPresence(false);
+        }
+    }, []);
```

Comment:
Maybe add apiClients to dependency array?

Reply:
Added apiClients to dependency array